app/controllers/finders_controller.rb

```diff
@@ -28,7 +28,7 @@ def show
           render 'finders/show-redirect'
         elsif finder.atom_feed_enabled?
           expires_in(ATOM_FEED_MAX_AGE, public: true)
-          @feed = AtomPresenter.new(finder)
+          @feed = AtomPresenter.new(finder, results)
```

Comment:
can we use `@results` here, otherwise it'll run through the `result_set_presenter` again.  (Or some other method of limiting the processing)

Reply:
Sure thing! Done :)